catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloadManager.java

```diff
@@ -190,6 +208,7 @@ private ResourceResponse startDownload(String downloadIdentifier,
                 retriever);
 
         ResourceResponse response = downloader.setupDownload(metacard, downloadStatusInfo);
+        response.getProperties().put("downloadIdentifier", downloadIdentifier);
```

Comment:
Didn't you have a `static String` constant for this? Was it `DOWNLOAD_ID` found within `DownloadInfo.java` and if it's not public should it be declared elsewhere too?